src/components/DashboardSiderbar.jsx

```diff
@@ -4,6 +4,7 @@ import PropertyGrid from './PropertyGrid';
 import MapPanel from './MapPanel';
 import SidebarMenu from './SidebarMenu';
 import LanguageSwitch from './LanguageTest';
+import NewReport from "./NewReport";
```

Comment:
As far as I can see we use a single quotation for the imports. We should configure the ESLint.